js/parts/Series.js

```diff
@@ -4081,6 +4081,10 @@ null,
         if (!series.xAxis) {
             return;
         }
+        // Don't invert series on inverted polar
+        if (series.isRadialSeries) {
+            inverted = false;
```

Comment:
Consider to use it as default option of serie `defaultCircularRadialOptions`

Reply:
I'm not sure if I understand this one but the isRadialSeries is a simple flag that is set on afterInit event to recognize series on inverted polar. It is used to prevent from inverting group of inverted polar series even though chart.inverted is set to true. I think it is unnecessary to place it in defaultCircularRadialOptions object, as it contains options for axis. What do you think @TorsteinHonsi ?

Reply:
1. We should not re-define function's parameters (like `inverted`)
2. I suggest to move this param to serie's init / default polar options.

Reply:
Ok, thanks for the further information. This options is already set on the event but it is `afterInit`. As for the modifying the `inverted` parameter, I've got rid of it, instead I added a simple check in `setInvert`.